packages/cmf/scripts/cmf-settings.i18n.js

```diff
@@ -0,0 +1,255 @@
+const path = require('path');
+const fs = require('fs');
+const jsonpath = require('jsonpath');
+const cloneDeep = require('lodash/cloneDeep');
+const difference = require('lodash/difference');
+const get = require('lodash/get');
+const intersection = require('lodash/intersection');
+const set = require('lodash/set');
+const mkdirp = require('mkdirp');
+
+const { getLogger } = require('./cmf-settings.utils');
+const { getJSON } = require('./getJSON');
+
+const JSON_PATH_EXPRESSION = '$..i18n';
+const DEFAULT_LOCALE = 'en';
+
+/**
+ * getPathFromPattern - get the path from the current pattern
+ *
+ * @param  {string} pattern   is the template string of the path with {{namespace}} and {{local}}
+ * @param  {string} namespace namespace to set
+ * @param  {string} locale    local to set
+ * @return {string}           path with replaced value
+ */
+function getPathFromPattern(pattern, namespace, locale) {
+	const replaceMap = {
```

Comment:
please check if pattenr contains {{namespace}} and {{local}} using the same regex (moved in a constant)